server/etc/default/pulp.logrotate

```diff
@@ -0,0 +1,8 @@
+/var/log/pulp/*log {
```

Comment:
It seems historically we've organized the config files in the path they are installed at under ``server/``. To keep with that style, could you move this to ``server/etc/logrotate.d/pulp``?